cluster/engine.go

```diff
@@ -620,6 +620,9 @@ func (e *Engine) updateContainer(c dockerclient.Container, containers map[string
 		}
 		// Convert the ContainerConfig from inspect into our own
 		// cluster.ContainerConfig.
+		if info.HostConfig != nil {
+			info.Config.HostConfig = *info.HostConfig
```

Comment:
Right, my question was about the RHS. I didn't realize the `HostConfig` inside info is a pointer, so my bad.